runtime/integration-tests/src/cases/lp/setup_evm.rs

```diff
@@ -0,0 +1,578 @@
+// Copyright 2024 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use super::*;
+
+/// Replicating Deployer.sol function `_rely`
+/// Source: https://github.com/centrifuge/liquidity-pools/blob/4d1d6f7cc5bf5022a83333697ee5040b620c4bdc/script/Deployer.sol#L75
+pub fn rely<T: Runtime>(evm: &mut <RuntimeEnv<T> as EnvEvmExtension<T>>::EvmEnv) {
+	/* NOTE: This rely is NOT needed as the LocalRouter is not permissioned
+	evm.call(
+		Keyring::Alice,
+		Default::default(),
+		names::ADAPTER,
+		"rely",
+		Some(&[Token::Address(evm.deployed(names::ROOT).address())]),
+	)
+	.unwrap(); */
+
```

Comment:
Should be then removed?

Reply:
Let's please keep this. Right now, we can only replicate the Solidity setup manually. This comment should signal that it is not necessary to perform this action at this point but that the author was aware of it.

Reply:
Ok, I see! Could we then add this message as a comment to it?

Reply:
Doesn't the current line suffice? 

> NOTE: This rely is NOT needed as the LocalRouter is not permissioned

Reply:
Thanks! I didn't catch the meaning at first without being involve on it